test/node/pipe.js

```diff
@@ -12,13 +17,19 @@ app.get("/", (req, res) => {
 });
 
 app.post("/", (req, res) => {
-  res.send(req.body);
+  if (process.env.EXPOSE_HTTP2){
+    res.set('content-type', 'application/json');
```

Comment:
Why is this needed?

Reply:
http/2 module have not full compatibility with http/1 module. http/2 imcommingHttpRequest does not have body property.